modules/tile-converter/src/i3s-converter/helpers/geometry-attributes.ts

```diff
@@ -267,8 +265,43 @@ function groupAttributesAndRangesByFeatureId(
     objIndex += 1;
   }
 
-  range.push(positions.length / POSITIONS_AND_NORMALS_PER_TRIANGLE - 1);
+  const attributes = concatenateAttributes(unifiedObjects);
+
```

Comment:
Are triangles only supported? Would it be necessary to check the type of objects?